docs/core/tools/project-json-to-csproj.md

```diff
@@ -472,7 +472,7 @@ See also [Files](#files).
   <!-- summary is not migrated from project.json, but you can use the <Description> property for that if needed. -->
   <PackageTags>machine learning;framework</PackageTags>
   <PackageReleaseNotes>Version 0.9.12-beta</PackageReleaseNotes>
-  <PackageIconUrl>http://numl.net/images/ico.png</PackageIconUrl>
+  <PackageIcon>http://numl.net/images/ico.png</PackageIconUrl>
```

Comment:
I think this shouldn't use a URL at all, based on @JamesNK suggestion in https://github.com/dotnet/docs/pull/19628

Reply:
#19746 might be related here too.

Reply:
What would the typical path-to-file be here then for an example?  Should we put just the file name?

Reply:
The examples I've seen were only file names.
But I dont know if PackageIconFullPath property is required or not.